_CoqProject

```diff
@@ -0,0 +1,10 @@
+# This top-level _CoqProject exists to make it easier to navigate
+# the Cava source and examples using the vscoq plug-in for vscode.
+-R cava/Cava Cava
+-Q third_party/coq-ext-lib/theories ExtLib
+-Q third_party/bedrock2/bedrock2/src/bedrock2 bedrock2
+-Q third_party/bedrock2/deps/coqutil/src/coqutil coqutil
+-Q monad-examples MonadExamples
+-Q monad-examples/xilinx XilinxExamples
+-Q arrow-examples ArrowExamples
+-Q arrow-lib Arrow
```

Comment:
Can you rebase on main? arrow-lib no longer exists on main branch

Reply:
Done.